packages/forms/src/text/FormikTextField.tsx

```diff
@@ -88,7 +88,7 @@ export const FormikTextField: ForwardRefExoticComponent<FormikTextFieldProps> =
           onChange={(event) => {
             onChange?.(event);
 
-            if (!event.defaultPrevented) {
+            if (!event.defaultPrevented || props.select) {
```

Comment:
why we need this props?

Reply:
@umrzoq-toshkentov when you try to select an option with `enter` it will not select the option because for Textfield select material is setting defaultPrevented `true`.

Reply:
let me know if my explanation was not satisfactory (-_-)`